### ISSUE TITLE
type for message_stats & next_seq_id

### ISSUE THREAD

**Issue by akutta** [michaelklishin/rabbit-hole#93]
deliver, deliver_get, publish, next_seq_id, etc should be int64 instead of int.  below payload causes fatal errors from json deserialization.

```json
{
    "owner_pid_details": {
      "name": "172.24.16.13:65493 -> 172.24.16.49:5672",
      "peer_port": 65493,
      "peer_host": "172.24.16.13"
    },
    "message_stats": {
      "ack": 2263340688,
      "ack_details": {
        "rate": 944.6
      },
      "deliver": 2263340695,
      "deliver_details": {
        "rate": 944.2
      },
      "deliver_get": 2263340695,
      "deliver_get_details": {
        "rate": 944.2
      },
      "publish": 2263340507,
      "publish_details": {
        "rate": 947.6
      }
    },
    "messages": 1,
    "messages_details": {
      "rate": 0.4
    },
    "messages_ready": 0,
    "messages_ready_details": {
      "rate": 0.0
    },
    "messages_unacknowledged": 1,
    "messages_unacknowledged_details": {
      "rate": 0.4
    },
    "policy": "",
    "exclusive_consumer_tag": "",
    "consumers": 1,
    "memory": 4862976,
    "backing_queue_status": {
      "q1": 0,
      "q2": 0,
      "delta": [
        "delta",
        "undefined",
        0,
        "undefined"
      ],
      "q3": 0,
      "q4": 0,
      "len": 0,
      "pending_acks": 1,
      "target_ram_count": "infinity",
      "ram_msg_count": 0,
      "ram_ack_count": 1,
      "next_seq_id": 2263340687,
      "persistent_count": 0,
      "avg_ingress_rate": 948.5090651251903,
      "avg_egress_rate": 948.5090651251903,
      "avg_ack_ingress_rate": 948.5090651251903,
      "avg_ack_egress_rate": 948.8090047473444
    },
    "status": "running",
    "name": "amq.gen-R6pjX7OAZ84E0_Xjt52CRg",
    "vhost": "/",
    "durable": false,
    "auto_delete": true,
    "arguments": {},
    "node": "rabbit@rmq-106"
  }
```

**Comment by michaelklishin** [comment]
@akutta feel free to submit a pull request.

**Comment by michaelklishin** [comment]
Fixed in #94, thank you @akutta for the contribution.
